notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    await atcs.rem.ataos.cmd_offset.set_start(z= -(z_offset_end + z_offset_step))
</pre><p>This might work... another option would be to get the current offset and compare it to the original offset, then just add the difference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>OK. so long as you checked it on the TTS it should be fine.</p>